flake8_dcos_lint/check_rules.py

```diff
@@ -0,0 +1,10 @@
+import re
+
+from collections import namedtuple
+
+CheckRegex = namedtuple("CheckRegex", "regex code reason")
+
+
+regex_rules = [
+    CheckRegex(re.compile(r'assert\s(\w+\.ok)$'), "D001", "Invalid assertion of type assert <symbol>.ok found.")
```

Comment:
I think it'd be worth putting this entire rationale in the comment for posterity. The current comment leaves out the context that this is about the `requests` API, for example. Anyone who runs into this error and tries to dig into it would probably appreciate the full explanation, and it's something that we can use if we ever revisit this rule or wonder why it's there.